make_beam/make_beam.c

```diff
@@ -310,11 +311,11 @@ int main(int argc, char **argv)
     
     // Create array for holding the raw data
     int bytes_per_file = opts.sample_rate * nstation * npol * nchan;
-    uint8_t *data;
-    //uint8_t *data1;
-    //uint8_t *data2;
+    uint8_t *data_read;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'data_read' can be reduced.](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4540729)